src/main/kotlin/com/openlattice/hazelcast/processors/organizations/UpdateOrganizationExternalDatabaseColumnEntryProcessor.kt

```diff
@@ -0,0 +1,33 @@
+package com.openlattice.hazelcast.processors.organizations
+
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.edm.requests.MetadataUpdate
+import com.openlattice.organization.OrganizationExternalDatabaseColumn
+import java.util.*
+
+data class UpdateOrganizationExternalDatabaseColumnEntryProcessor(val update: MetadataUpdate) :
+        AbstractRhizomeEntryProcessor<UUID, OrganizationExternalDatabaseColumn, OrganizationExternalDatabaseColumn?>() {
+
+    override fun process(entry: MutableMap.MutableEntry<UUID, OrganizationExternalDatabaseColumn>): OrganizationExternalDatabaseColumn? {
```

Comment:
I think the return value shouldn't be nullable if the entry value is not nullable